native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1314,6 +1314,11 @@ struct InputAssemblerInfo {
     AttributeList attributes;
     BufferList vertexBuffers;
     Buffer *indexBuffer{nullptr}; // @ts-nullable
+    uint32_t vertexCount{0};
+    uint32_t firstVertex{0};
+    uint32_t indexCount{0};
+    uint32_t firstIndex{0};
+    int32_t vertexOffset{0};
```

Comment:
Are these members appropriate? @bluesky013

Reply:
The InputAssembler is recommended to be used only as a container for vertex buffers (VBs) and index buffers (IBs), while draw call parameters should be independent of the InputAssembler. The usage of ```draw(ia)``` will be deprecated in the future.

Usage similar to:
```cpp
bindInputAssembler()
setXXX(vp/scissor)
draw(param1)
setXXX(vp/scissor)
draw(param2)
setXXX(vp/scissor)
draw(param3)
```

Reply:
The current InputAssembler contains a _drawInfo. We cannot just initialize the _drawInfo data from InputAssemblerInfo. Can we wait until _drawInfo is removed from InputAssembler and delete the draw info data of InputAssemblerInfo as well? @blu

Reply:
Refactor drawInfo from InputAssembler later